src/main/java/bio/terra/service/snapshot/SnapshotDao.java

```diff
@@ -495,12 +505,26 @@ public void updateSnapshotTableRowCounts(Snapshot snapshot, Map<String, Long> ta
 
     private static class SnapshotSummaryMapper implements RowMapper<SnapshotSummary> {
         public SnapshotSummary mapRow(ResultSet rs, int rowNum) throws SQLException {
+            List<StorageResource> storageResources = new ArrayList<>();
+            Array rs_array = rs.getArray("storage");
+            String result_string = rs_array.toString();
+            JSONArray storageObjects = new JSONArray(result_string);
+            for (int i = 0; i < storageObjects.length(); i++) {
+                var sr = storageObjects.getJSONObject(i);
+                var storageResource = new StorageResource()
+                        .cloudPlatform(CloudPlatform.valueOf(sr.getString("cloudPlatform")))
+                        .cloudResource(GoogleCloudResource.valueOf(sr.getString("cloudResource")))
+                        .region(GoogleRegion.valueOf(sr.getString("region")));
```

Comment:
The `storage` field in the response uses the `StorageResourceModel` schema, which doesn't include the `datasetId`.

Reply:
I *think* you might be able to do this in one line (because the response looks like the `StorageResource` pojo fields:
```
List<StorageResource> storageResources = objectMapper.readValue(rs.getString("storage"), new TypeReference<List<StorageResource>>{});
```

or if that doesn't work:
```
List<StorageResource> storageResources = objectMapper.readValue(rs.getArray("storage").toString(), new TypeReference<List<StorageResource>>{});
```

I think postgres returns json as Strings

Reply:
@tlangs I ended up including the `datasetId` here so that should be less confusing!